.gitignore

```diff
@@ -294,3 +294,4 @@ AddOns/AMCBanking365Fundamentals/test/app.json
 AddOns/AMCBanking365Fundamentals/test/AppResources/Mock request  response/App.config
 AddOns/AMCBanking365Fundamentals/app/app.json
 AddOns/AMCBanking365Fundamentals/app/app.json
+**/.netpackages/**
```

Comment:
Unrelated change. Please revert.

Reply:
Done. Can I open another related PR for this? all 728 .net assemblies are shown as changes...

Reply:
Sure you can :-) But we prefer to separate issues from each other. Makes PR management (e.g. backporting to earlier releases) SO much easier :-)